test/turbopack-build-tests-manifest.json

```diff
@@ -15604,68 +15604,68 @@
       "runtimeError": false
     },
     "test/production/app-dir/actions-tree-shaking/basic/basic.test.ts": {
-      "passed": [
+      "passed": [],
+      "failed": [
         "actions-tree-shaking - basic should not have the unused action in the manifest"
```

Comment:
We could just set `treeShaking: true` for these test cases

Reply:
Oh.... I'll send another PR tomorrow